tests/devices/eiger/test_eiger.py

```diff
@@ -0,0 +1,347 @@
+# import logging
+# import struct
+# from typing import Optional
+
+import pytest
+
+# import aiohttp
+from aiohttp import web
+from mock import MagicMock, Mock
+from mock.mock import create_autospec
+
+# from tickit.core.device import DeviceUpdate
+# from tickit.core.typedefs import SimTime
+from tickit.devices.eiger.eiger import Eiger, EigerAdapter
+from tickit.devices.eiger.eiger_settings import EigerSettings
+from tickit.devices.eiger.eiger_status import EigerStatus, State
+
+# # # # # Eiger Tests # # # # #
+
+
+@pytest.fixture
+def eiger() -> Eiger:
+    return Eiger()
+
+
+def test_eiger_constructor():
+    Eiger()
+
+
+@pytest.mark.asyncio
+async def test_eiger_initialize(eiger):
+    await eiger.initialize()
+
+    assert eiger.get_state() == State.IDLE
```

Comment:
Typically, the asserted value is put on the left hand side (not sure why, but it's convention)

Reply:
I've seen this convention too, but not sure where it comes from. Have you seen this written down anywhere? I prefer `assert get_value() == expected_value` and write this in my tests